ClinGen/clincoded
# Connects to #1138. Calculate score for evidence of Individual, Experimental and Case Control.

**Know issues:**
1. The scoring for Experimental is different from Individual and Case Control. So its currently state is incorrect and I will work on it separately.

**Steps to test:**
1. Login and create a new GDM using one of the following 3 types of MOI: Autosomal Dominant, Autosomal Recessive and X-Linked.
2. Individually create new evidence for Family, Experimental and Case-Control. For the Family evidence, add a Proband and associate it with 1 or 2 variants.
3. Score the 3 different types of evidence respectively.
4. Upon being redirected back to the Curation-Central page, click on any of the "Edit" links and expect to see the previously saved score data on the subsequent form pages.
5. Return the Curation-Central page, click on any of the "View/Score" links and expect to see the previously saved score data on the subsequent "viewer" pages.
6. Logout and login as a different user. Visit the same GDM and click on the any of the "View/Score" links and expect to see the curator-saved score data on the subsequent "viewer" pages.
7. Score the 3 different types of evidence respectively. Expect to see the scores being saved and appended to the pre-existing scores.
